server/directory/views.py

```diff
@@ -21,4 +22,17 @@ def get(self, request):
             )
 
         serializer = DirectoryMemberSerializer(members, many=True)
-        return Response(serializer.data)
\ No newline at end of file
+        return Response(serializer.data)
+
+class MemberDirectoryView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        # get id from path parameter
```

Comment:
nit: do you need this comment? The code itself shows what it does.

Reply:
I specifically wrote this because i wanted to differentiate between the id being from a query vs in the url itself.

Reply:
I see, okay.